storage/src/adb/current.rs

```diff
@@ -311,7 +269,16 @@ impl<
         // commit op that will be appended.
         self.raise_inactivity_floor(self.any.uncommitted_ops + 1)
             .await?;
+
+        // Sync the log and process the updates to the MMR in parallel.
+        let log_fut = async { self.any.log.sync().await.map_err(Error::Journal) };
+        let mmr_fut = async {
+            self.any.mmr.process_updates(&mut self.any.hasher);
```

Comment:
This change doesn't really impact recovery at all (it is just an in-memory thing we were previously doing serially?

Reply:
yes correct, this snuck into this PR a bit prematurely, but I'm starting to do this in all the databases (update the MMR in parallel with the sync operation rather than doing them sequentially).  I expect with large enough commits this could be non-negligible?

Reply:
Yeah, I agree